src/main/java/net/dunice/IdenticalPairs.java

```diff
@@ -0,0 +1,27 @@
+package net.dunice;
+
+import java.util.*;
+
+public class IdenticalPairs {
+    public static void main(String[] args) {
+        int[] myArray = {1, 2, 3, 3, 3, 4, 5, 6, 2, 4, 5};
```

Comment:
в логике есть проблемы, считает пары не верно
```suggestion
        int[] myArray = {1, 2, 3, 3, 3, 4, 5, 6, 2, 4, 5};
```
здесь пар больше, чем приложение считает (здесь их вроде 6 пар)